lib/model/rofolder.go

```diff
@@ -47,7 +37,7 @@ func (f *roFolder) Serve() {
 	defer l.Debugln(f, "exiting")
 
 	defer func() {
-		f.scan.timer.Stop()
+		f.scan.Timer().Stop()
```

Comment:
@calmh is it correct to have a getter here?

Or was the intent `f.scan.Stop()` to be equivalent of `f.scan.timer.Stop()`?